phpunit.xml.dist

```diff
@@ -27,6 +27,9 @@
     <testsuite name="executor-tests">
         <directory>Tests/Executor</directory>
     </testsuite>
+    <testsuite name="abs-tests">
+        <file>Tests/Runner/DataLoaderABSTest.php</file>
```

Comment:
pockej, to je blbost ne? to se prece spusti v ramci runner-tests

Reply:
Jo, spusti se to 2x. Jednou s "normlanym" tokenom a jednou s ABS. Co sice uplne nevadi, ale potom ten nazov tej triedy toho testu nedava uplne zmysel.

Reply:
Hodim tam exclude alebo premenujem tu triedu na `DataLoaderFilesTest` napr?

Reply:
no ta metoda testLoadInputData je stejne v obou dataloader testech, takze se ted vlastne spusti 3x :)
Takze co takhle - vyhdit ji z DataLoaderTest a DataLoaderABSTest vyhodit do zvlasniho adresare backend-specific-tests a ten pak bude v phpunit.xml 2x - jednou s s3 tokenem a jednou s abs tokenem ?

Reply:
Hele, neni uplne stejna, ja som do tejto ABS varianty pridal "files", predtym sa tam testovali len "tables".
Ale kludne to mozme spravit tak ako pises. Ok.

Reply:
aha, jo to jsem zase koukal jinam